src/main/java/com/example/employee_management/service/EmployeeService.java

```diff
@@ -0,0 +1,47 @@
+package com.example.employee_management.service;
+
+import com.example.employee_management.Employee;
+import com.example.employee_management.exception.EmployeeAlreadyAddedException;
+import com.example.employee_management.exception.EmployeeNotFoundException;
+import com.example.employee_management.exception.EmployeeStorageIsFullException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeService {
+    private final List<Employee> employees = new ArrayList<>();
+    private static final int MAX_EMPLOYEES = 100;
+
+    public void addEmployee(String firstName, String lastName) {
+        Employee newEmployee = new Employee(firstName, lastName);
+
+        if (employees.contains(newEmployee)) {
+            throw new EmployeeAlreadyAddedException();
+        }
+
+        if (employees.size() >= MAX_EMPLOYEES) {
+            throw new EmployeeStorageIsFullException();
+        }
+
+        employees.add(newEmployee);
+    }
+
+    public void removeEmployee(String firstName, String lastName) {
+        Employee employeeToRemove = new Employee(firstName, lastName);
+
+        if (employees.remove(employeeToRemove)){
```

Comment:
Логика должна быть обратной - если НЕ удалось удалить сотрудника, то выкидываем исключение
```suggestion
        if (!employees.remove(employeeToRemove)){
```